src/services/fhirService.ts

```diff
@@ -340,6 +349,15 @@ export const prepareSearchParameters = (params: Params) => {
     parameters.exclude_tags = excludeTags;
   }
 
+  if (params.fhirVersion) {
+    if (useFallbackParams) {
+      parameters.tags.push(taggingUtils.buildFallbackTag(tagKeys.fhirVersion, params.fhirVersion));
+    } else {
+      parameters.tags.push(taggingUtils.buildTag(tagKeys.fhirVersion, params.fhirVersion));
+    }
+    delete params.fhirVersion;
```

Comment:
This line changes the parameters handed in and seems to be pretty unnecessary.

Reply:
Good point, I actually meant the `parameters`.